layouts/ucf-post-list-material.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
A new shortcode layout falls outside of the scope of this PR--please remove the "material" layout addition in this PR and submit it as its own PR if you feel it's an appropriate inclusion in this plugin for everyone using it.  Thanks!